appserver/tests/payara-samples/samples/microprofile-config-alias-properties/src/test/java/fish/payara/samples/microprofile/config/alias/MicroProfileConfigAliasTest.java

```diff
@@ -0,0 +1,105 @@
+ * only the GPL Version 2, indicate your decision by adding "[Contributor]
+ * elects to include this software in this distribution under the [CDDL or GPL
+ * Version 2] license."  If you don't indicate a single choice of license, a
+ * recipient has the option to distribute your version of this file under
+ * either the CDDL, the GPL Version 2 or to extend the choice of license to
+ * its licensees as provided above.  However, if you add GPL Version 2 code
+ * and therefore, elected the GPL Version 2 license, then the option applies
+ * only if the new code is made subject to such option by the copyright
+ * holder.
+ */
+package fish.payara.samples.microprofile.config.alias;
+
+import com.gargoylesoftware.htmlunit.TextPage;
+import com.gargoylesoftware.htmlunit.WebClient;
+import fish.payara.samples.CliCommands;
+import fish.payara.samples.NotMicroCompatible;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.io.File;
+import java.net.URL;
+import java.nio.file.Paths;
+
+/**
+ * @author Andrew Pielage <andrew.pielage@payara.fish>
+ */
+@RunWith(Arquillian.class)
+@NotMicroCompatible
+public class MicroProfileConfigAliasTest {
+
+    @ArquillianResource
+    private URL url;
+
+    @Deployment(testable = false)
+    public static WebArchive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class, "microprofile-config-alias.war")
+                .addPackage("fish.payara.samples.microprofile.config.alias")
+                .addAsManifestResource(new File("src/main/resources/META-INF/microprofile-config-alias.properties"));
+    }
+
+    @BeforeClass
+    public static void createPasswordAlias() {
+        CliCommands.payaraGlassFish("create-password-alias", "-W", Paths.get("").toAbsolutePath() + "/src/test/resources/passwordfile.txt", "wibbles");
+
+        // Deployment actually happens before @BeforeClass, and the PasswordAlias config source requires an application refresh to update, so disable and enable it
+        CliCommands.payaraGlassFish("set", "servers.server.server.application-ref.microprofile-config-alias.enabled=false");
+        CliCommands.payaraGlassFish("set", "servers.server.server.application-ref.microprofile-config-alias.enabled=true");
+    }
+
+    @AfterClass
+    public static void deletePasswordAlias() {
+        CliCommands.payaraGlassFish("delete-password-alias", "wibbles");
+    }
+
+    @Test
+    public void testAliasSubstitution() throws Exception {
+        TextPage page = new WebClient().getPage(url + "ConfigServlet");
+        System.out.println(page.getContent());
```

Comment:
Is this intentionally left `println` ?

Reply:
Yes, it lets you see what the page actually returns without using a debugger. It also means it prints it on the terminal running the test rather than in the server log.
Or are you referring specifically to `print` vs. `println`?

Reply:
Just wanted make sure this is intentionally present.